lake/commit/snapshot.go

```diff
@@ -84,6 +84,14 @@ func (s *Snapshot) Select(span nano.Span) Segments {
 	return segments
 }
 
+func (s *Snapshot) SelectAll() Segments {
+	var segments Segments
+	for _, seg := range s.segments {
+		segments = append(segments, seg)
```

Comment:
```suggestion
		segments.Append(seg)
```

Reply:
That doesn't work. Maybe we should get rid of Segments as you suggested.